### ISSUE TITLE
[Feature] Print SQL statements to fix false m-n relations (requires data migration)

### ISSUE THREAD

**Issue by nikolasburk** [prisma/upgrade#27]
The exact problem + SQL workaround are described here: https://www.prisma.io/docs/guides/upgrade-from-prisma-1/schema-incompatibilities-postgres#all-non-inline-relations-are-recognized-as-m-n

Note that this requires a data migration, so we should figure out a good way how to integrate it in the Upgrade CLI (e.g. as an additional step after all the SQL that only perform Prisma-1-compatible changes was printed). 

Internal link: https://www.notion.so/prismaio/Limitations-of-Introspection-of-Prisma-1-schemas-Workarounds-24cfc8cb8cc3402281ee708f66339d42#21eeaef29947443eb5cb7a029a29bed8

**Comment by nikolasburk** [comment]
For the record, that doesn't seem to work for my sample data model yet:

```graphql
type User {
  id: ID! @id
  email: String @unique
  name: String!
  role: Role! @default(value: CUSTOMER)
  jsonData: Json 
  posts: [Post!]!
}
type Post {
  id: ID! @id
  createdAt: DateTime! @createdAt
  updatedAt: DateTime! @updatedAt
  title: String!
  content: String
  published: Boolean! @default(value: false)
  author: User @relation(link: TABLE)
}

```

**Comment by matthewmueller** [comment]
This should be fixed in 0.0.23
